.github/actions/check-skip/action.yaml

```diff
@@ -0,0 +1,144 @@
+name: 'Check skips'
+description: 'Check whether some workflows should be skipped'
+inputs:
+  ignore-changes:
+    required: false
+    description: |
+      Directories and files to ignore when checking changes to determine whether functional
+      tests should be skipped. This is an optional input field and it accepts a regex string.
+      The paths should be relative to the repo's root directory.
+      Examples:
+      - "^(README.md|doc/|tests/unit/)"
+    default: "^$"
+  check-lint-unit-skip:
+    required: false
+    description: "Whether to check the possibility of skipping lint and unit tests"
+    default: false
+  check-func-skip:
+    required: false
+    description: "Whether to check the possibility of skipping functional tests"
+    default: false
+outputs:
+  skip-lint-unit-tests:
+    description: "Whether to skip lint and unit tests"
+    value: ${{ steps.skip-lint-unit-tests.outputs.skip }}
+  skip-func-tests:
+    description: "Whether to skip functional tests"
+    value: ${{ steps.skip-func-tests.outputs.skip }}
+runs:
+  using: "composite"
+  steps:
+    - name: Get latest workflow info
+      id: latest-workflow
+      env:
+        CURRENT_BRANCH: ${{ github.event.pull_request.head.ref }}
+      shell: bash
+      run: |
+        function get_last_workflow() {
+          # get all previus workflows for "test_full.yml" file and current branch
+          _workflows=$(curl -H "Accept: application/vnd.github.v3+json" "$GITHUB_API_URL/repos/$GITHUB_REPOSITORY/actions/workflows/pr.yaml/runs?branch=$CURRENT_BRANCH")
```

Comment:
I think it would be more useful if we define name of workflow file as input argument. The "pr.yaml" cloud be default value.